modules/bump-version.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+if [[ $(__write_version) == false ]]; then
+    return 0
+fi
```

Comment:
By returning here, not allowing the execution to continue, bumping of the version will be completely disabled. I don't think that's what we want. I think we only want to prevent the version being read from and written to a file.

As an alternative, you could truncate the value of `$VERSION_FILE` if `$(__write_version)` is `false` (making it empty). Then the `semverbump.sh` could skip reading from the version file if it's empty. It would still lookup the version based on git tags (in fact, it does that first, and only uses the file if no suitable tags were found).

The change you made to `write-version.sh` already makes sure the version file won't be written to.